bundles/org.openhab.binding.homekit/README.md

```diff
@@ -0,0 +1,127 @@
+# HomeKit Binding
+
+This binding allows pairing with HomeKit accessories and **imports** their services as channel groups and their respective service- characteristics as channels.
+Do not confuse this with the other HomeKit **integration** (https://www.openhab.org/addons/integrations/homekit/) which **exports** openHAB Items to a HomeKit controller.
```

Comment:
I guess the "other" is categorized as a "system integration", so we can refer to that and skip "other"?
```suggestion
Do not confuse this with the HomeKit **system integration** (https://www.openhab.org/addons/integrations/homekit/) which **exports** openHAB Items to a HomeKit controller.
```